Yoast/Yoast-SEO-for-TYPO3
# [BUGFIX] Removed non-existing / unnecessary CSS file within PageOverviewWidget

## Summary

This PR can be summarized in the following changelog entry:

* Non-existing css file `extended-list-widget.min.css` from `PageOverviewWidget`

## Relevant technical choices:

* Removed the whole `getCssFiles` method and the `AdditionalCssInterface`

## Test instructions

This PR can be tested by following these steps:

* Pull branch and check if the dashboard widgets are still working

## Quality assurance


Fixes #537
